src/tree/logic-app/LogicAppTriggerTreeItem.ts

```diff
@@ -19,6 +19,10 @@ export class LogicAppTriggerTreeItem implements IAzureTreeItem {
         return "azureLogicApps.openTriggerInEditor";
     }
 
+    public get callbackUrl(): Promise<WorkflowTriggerCallbackUrl> {
```

Comment:
(nit) Instead of a property getter, can we change this to a `getCallbackUrl` function since it returns a promise?